js/lib/ui.js

```diff
@@ -467,6 +442,5 @@ elgg.ui.initAccessInputs = function () {
 };
 
 elgg.register_hook_handler('init', 'system', elgg.ui.init);
-elgg.register_hook_handler('init', 'system', elgg.ui.initDatePicker);
```

Comment:
We can't throw deprecation errors if we gonna rewire them. Alternative to add an extra selector/data attr in input/date view, and log a deprecation warning if it's not there.